### PR TITLE
Replaces 'walkthrough' with 'solution pattern' in hyperlinks

### PR DESCRIPTION
**Pull request by briandooley**
@pwright Review pur-lease?

### CODE REVIEW

File: modules/ROOT/pages/_partials/gs-publishing-walkthroughs-proc.adoc [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 The home page of {webapp} lists all the {walkthrough-name}s from all the Git repositories you are subscribed to. Any user can add {walkthrough-name}s to your cluster.
```
Comment (nebelsprachen) [inline modules/ROOT/pages/_partials/gs-publishing-walkthroughs-proc.adoc]: Slightly silly suggestion, but I think there might be too many instances of 'all'

```suggestion
The home page of {webapp} lists the {walkthrough-name}s from all the Git repositories you are subscribed to. Any user can add {walkthrough-name}s to your cluster.
```

File: modules/ROOT/pages/_partials/gs-writing-walkthroughs-proc.adoc [hunk 2]
```diff
@@ -494,7 +494,7 @@ For example, to use the `CakePHP + MySQL (Ephemeral)` service in your custom {wa
 }
 ----
 +
-. Add your {walkthrough-name} to the cluster as described in xref:gs-publishing-walkthroughs-proc[Adding walkthroughs].
+. Add your {walkthrough-name} to the cluster as described in xref:gs-publishing-walkthroughs-proc[Adding {walkthrough-name}s].
 . Navigate the {walkthrough-name} to trigger the provisioning of the custom service, for example, the `cakephp-mysql-example` service.
 . Log in to the OpenShift console.
 . Navigate to the custom project in OpenShift.The project is named using the pattern `<username>-<walkthrough-id>`.
```
Comment (nebelsprachen) [inline modules/ROOT/pages/_partials/gs-writing-walkthroughs-proc.adoc:500]: Has the <walkthrough-id> also been renamed to solution pattern?

**Review by nebelsprachen — APPROVED** [review]
Looks great Brian!

File: modules/ROOT/pages/_partials/gs-writing-walkthroughs-proc.adoc (hunk 2 above)
Comment (briandooley) [reply modules/ROOT/pages/_partials/gs-writing-walkthroughs-proc.adoc:500]: Yep.  It has.

**Pull request merged**